from base.exceptions import CommandError

class BookValidationError(CommandError):
    pass

